ui/glide/src/main/java/de/danoeh/antennapod/ui/glide/ResizingOkHttpStreamFetcher.java

```diff
@@ -118,6 +119,14 @@ public void onDataReady(@Nullable InputStream data) {
                         e.printStackTrace();
                         callback.onLoadFailed(fileNotFoundException);
                     }
+                } finally {
+                    if (outputStream != null) {
+                        try {
+                            outputStream.close();
+                        } catch (Exception e) {
+                            e.printStackTrace();
+                        }
+                    }
```

Comment:
Can be simplified using `IOUtils.closeQuietly`. Also, this change doesn't really fit together with the other changes that his PR does.

Reply:
IOUtils.copy(data, outputStream);
the upper codes maybe throws an IOException.then the outputStream will not be closed.